3570/3/rating_places/app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,10 @@
+require_relative 'application_controller'
+
+class CommentsController < ApplicationController
+  post '/places/:id/comments' do
+    @comment = current_user.comments.create(rating: params[:rating].to_i,
```

Comment:
I think you create a comment for the place. So, please, use `place.comments.create` instead of `current_user.comments.create`

Reply:
And I ask you again. Why the hell you need this instance variable?